lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -20,14 +20,14 @@ blockchain_configuration:
   block0_consensus: genesis_praos
 
   # Number of slots in each epoch
-  slots_per_epoch: 500
+  slots_per_epoch: 3
```

Comment:
3 seems like.. a bit excessive :/ but I have no strong argument to favor `10` or `500` more than `3` ...

Reply:
I think being excessive here is the only way we can make the test reliably succeed in a short enough time.